ehrapy/api/encode/encode.py

```diff
@@ -494,6 +542,134 @@ def _add_categoricals_to_uns(ann_data: AnnData, categorical_names: List[str]) ->
             elif var_name in categorical_names:
                 ann_data.uns["original_values_categoricals"][var_name] = ann_data.X[::, idx : idx + 1]
 
+    @staticmethod
+    def _extract_autodetect_and_encoding_modes(
```

Comment:
The name doesn't really tell me what it is supposed to do. At least I have troubles with it.